### ISSUE TITLE
error message directs user to bugzilla.xamarin.com

### ISSUE THREAD

**Issue by jrr** [mono/mono#9413]
I received this warning today:

`Microsoft.Common.Mono.After.targets(37, 9): [MSB3912] INTERNAL WARNING: Could not find the replacement assembly (System.Globalization.Extensions.dll) for the Windows specific reference /Library/Frameworks/Mono.framework/Versions/5.12.0/lib/mono/xbuild/Microsoft/Microsoft.NET.Build.Extensions/net461/lib/System.Globalization.Extensions.dll. Please file a bug report at https://bugzilla.xamarin.com . `

That bugzilla tells me:

>  Xamarin Bugzilla is no longer open for new bug reports. Please use the new locations below instead.

I'm not really concerned about resolving the warning right now, but it would be nice if the error message would direct the user to the correct issue tracker.

**Version Used**:

```
Mono JIT compiler version 5.12.0.226 (2018-02/9824e260f56 Mon Apr 30 18:45:52 EDT 2018)
Copyright (C) 2002-2014 Novell, Inc, Xamarin Inc and Contributors. www.mono-project.com
	TLS:           normal
	SIGSEGV:       altstack
	Notification:  kqueue
	Architecture:  amd64
	Disabled:      none
	Misc:          softdebug 
	Interpreter:   yes
	LLVM:          yes(3.6.0svn-mono-master/8b1520c8aae)
	GC:            sgen (concurrent by default)
```

**Comment by DustinCampbell** [comment]
After moving OmniSharp to target net461, we're getting this on build as well: https://github.com/OmniSharp/omnisharp-roslyn/issues/1256#issuecomment-410336175

**Comment by radical** [comment]
This warning has been removed completely now.

**Comment by galvesribeiro** [comment]
@radical not sure this error was removed...

I'm on OSX using VSCode/OmniSharp and at the same solution I have `netcoreapp2.1` and `net461` projects.

**Comment by DustinCampbell** [comment]
@galvesribeiro : I believe this was fixed after Mono 5.12.0.

**Comment by DustinCampbell** [comment]
In a newer release.
